_data/reviews.yml

```
@@ -347,4 +384,43 @@
       date-f: 23 June 2025
       date: 20250623
       name: Anonymous
+      type: Ad-Hoc
+    - score: 5
+      feedback: |
+        I'm so grateful to be connected with Lidiane. She has so many tips to share, and her advice has been incredibly relevant and helpful for my situation. She really took the time to understand my concerns and made me feel heard and supported. I deeply appreciate her warmth, openness, and the way she uplifts and empowers other women!
+      date-f: 30 June 2025
+      date: 20250630
+      name: Anonymous
+      type: Ad-Hoc
+
+- index: 20 # Andrew King
+  feedback:
+    - score: 5
+      feedback: |
+        It was great to hear experiences, as I learn and reflect about mines. Amazing experience highly recommend it.
+      date-f: 9 July 2025
+      date: 20250709
+      name: Anonymous
```

This mentee is happy to share their name:
```suggestion
      name: Shirley
```